tests/conftest.py

```diff
@@ -6,7 +6,8 @@
 
 def pytest_configure(config):
     pytest.ssh_config = SimpleNamespace(
-        TEST_SSH=False,
+        TEST_SSH=True,
+        PASSWORD_FILE=r"C:\Users\Joe\temp_pass.txt",  # don't store this on github!
```

Comment:
Should authentication not be carried out with ssh keys not passwords?

Reply:
Yes actually, for nearly all tests this is okay. There is a test for setting up the SSH key-pair, which is why I added this. I thoguht I could get away with it as a temp workaround, but having already accidently pushed a password to github and had to change a load of passwords because of it, I now think its a terrible idea 🥲 , at least to have the file in the repo. Maybe it is safest just to scrap the test for SSH key pair setup, and rely on the keys for all over tests.